src/components/business-lookup/BusinessLookup.vue

```diff
@@ -2,55 +2,70 @@
   <div id="business-lookup">
     <div v-if="state !== States.SUMMARY">
       <v-autocomplete
-        filled no-filter append-icon="" return-object
+        v-model="selectedBusiness"
+        v-model:search-input="searchField"
+        filled
+        no-filter
+        append-icon=""
+        return-object
         class="mt-5"
         autocomplete="chrome-off"
         menu-props="{ maxHeight: 380 }"
         label="Business or Corporation Name or Incorporation Number"
         item-text="identifier"
-        v-model="selectedBusiness"
         :name="Math.random()"
         :rules="showErrors ? businessLookupRules: []"
         :items="searchResults"
         :loading="state == States.SEARCHING"
-        :search-input.sync="searchField"
         :hide-no-data="state != States.NO_RESULTS"
       >
         <!-- Empty selection slot will stop re-triggering of searchField @Watch -->
-        <template v-slot:selection=""></template>
+        <template #selection="" />
 
-        <template v-slot:no-data>
+        <template #no-data>
           <v-list-item>
             <div>No matches found.</div>
           </v-list-item>
         </template>
 
-        <template v-slot:item="{ item }">
+        <template #item="{ item }">
```

Comment:
I should test this, too.

Reply:
ESLint changed this automatically based on its presets/plugins, so not "necessary" but gets done anyway. Thanks, Kial.